<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutUI</class>
 <widget class="QWidget" name="AboutUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/about/angles_flat.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;SasView&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="lblVersion">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblAbout">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="cmdLinkUT">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/utlogo.png</normaloff>:/about/utlogo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="cmdLinkUMD">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/umd_logo.png</normaloff>:/about/umd_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="cmdLinkNIST">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/nist_logo.png</normaloff>:/about/nist_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>110</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="cmdLinkESS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/ess_logo.png</normaloff>:/about/ess_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="cmdLinkISIS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/isis_logo.png</normaloff>:/about/isis_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="cmdLinkDELFT">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/tudelft_logo.png</normaloff>:/about/tudelft_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>128</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="cmdLinkILL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/ill_logo.png</normaloff>:/about/ill_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>54</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="cmdLinkANSTO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/ansto_logo.png</normaloff>:/about/ansto_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="cmdLinkDIAMOND">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/dls_logo.png</normaloff>:/about/dls_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>177</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="cmdLinkBAM">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/bam_logo.png</normaloff>:/about/bam_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>66</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="cmdLinkSNS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/about/ornl_logo.png</normaloff>:/about/ornl_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>96</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>593</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="cmdOK">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="lblText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This work originally developed as part of the DANSE project funded by the NSF under grant DMR-0520547, and currently maintained by UTK, NIST, UMD, ORNL, ISIS, ESS, ILL, ANSTO, TU Delft, DLS, and the scattering community.&lt;/p&gt;&lt;p&gt;SasView also contains code developed with funding from the EU Horizon 2020 programme under the SINE2020 project (Grant No 654000).&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmdLinkUT</tabstop>
  <tabstop>cmdLinkUMD</tabstop>
  <tabstop>cmdLinkNIST</tabstop>
  <tabstop>cmdLinkESS</tabstop>
  <tabstop>cmdLinkISIS</tabstop>
  <tabstop>cmdLinkDELFT</tabstop>
  <tabstop>cmdLinkILL</tabstop>
  <tabstop>cmdLinkANSTO</tabstop>
  <tabstop>cmdLinkDIAMOND</tabstop>
  <tabstop>cmdLinkBAM</tabstop>
  <tabstop>cmdLinkSNS</tabstop>
  <tabstop>cmdOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
